### PR TITLE
[Reopened] Rancher 839

### PR DESCRIPTION
**Pull request by guram-jalaghonia**
Rancher 839
Draft Version
Job Works But Need Team Suggestions as well for the Good Style 

https://issues.folio.org/browse/RANCHER-839

### CODE REVIEW

File: pipelines/folioRancher/folioScheduledTesting/runNightlySchemaComparison/Jenkinsfile [hunk 1]
```diff
@@ -1 +1,56 @@
-//TBD
+#!groovy
+
+import org.jenkinsci.plugins.workflow.libs.Library
+import org.folio.Constants
+
+@Library('pipelines-shared-library@RANCHER-839') _
```
Comment (eldiiar-duishenaliev) [inline pipelines/folioRancher/folioScheduledTesting/runNightlySchemaComparison/Jenkinsfile:6]: Don't forget to switch to 741 branch.

File: pipelines/folioRancher/folioScheduledTesting/executeSprintBasedDataMigration/Jenkinsfile [hunk 2]
```diff
@@ -1 +1,56 @@
...
+        try {
+          build job: Constants.JENKINS_JOB_DATA_MIGRATION,
+            parameters:
+              [string(name: 'FOLIO_REPOSITORY', value: FOLIO_REPOSITORY),
+               string(name: 'FOLIO_BRANCH_SRC', value: FOLIO_BRANCH_SRC),
+               string(name: 'FOLIO_BRANCH_DST', value: FOLIO_BRANCH_DST),
+               string(name: 'BACKUP_NAME', value: BACKUP_NAME),]
+        } catch (Exception new_ex) {
+          slackNotifications.sendPipelineFailSlackNotification("#folioschemacompare")
+          throw new Exception("Execution Of Schema Comparison Job Failed: " + new_ex)
```
Comment (eldiiar-duishenaliev) [inline pipelines/folioRancher/folioScheduledTesting/executeSprintBasedDataMigration/Jenkinsfile:51]: requires .getMessage() method inclusion.

File: pipelines/folioRancher/folioScheduledTesting/runNightlySchemaComparison/Jenkinsfile [hunk 3]
```diff
@@ -1 +1,56 @@
...
+
+properties([
+  //pipelineTriggers([cron('H 0 * * *')]),  # need to implement cron time
+  buildDiscarder(logRotator(numToKeepStr: '30')),
+  disableConcurrentBuilds(),
+  parameters([
+    string(name: 'FOLIO_REPOSITORY', defaultValue: 'platform-complete', description: "Repository where to build"),
+    string(name: 'FOLIO_BRANCH_SRC', defaultValue: 'R1-2023', description: "platform-complete source branch for the comparison"),
```
Comment (eldiiar-duishenaliev) [inline pipelines/folioRancher/folioScheduledTesting/runNightlySchemaComparison/Jenkinsfile:14]: Should it be snapshot branch specified over here?

File: pipelines/folioRancher/folioScheduledTesting/executeSprintBasedDataMigration/Jenkinsfile [hunk 4]
```diff
@@ -1 +1,56 @@
...
+
+@Library('pipelines-shared-library@RANCHER-839') _
+
+properties([
+  //pipelineTriggers([cron('H 0 * * *')]),  # need to implement cron time
+  buildDiscarder(logRotator(numToKeepStr: '30')),
+  disableConcurrentBuilds(),
+  parameters([
+    string(name: 'FOLIO_REPOSITORY', defaultValue: 'platform-complete', description: "Repository where to build"),
+    string(name: 'FOLIO_BRANCH_SRC', defaultValue: 'R1-2023', description: "platform-complete source branch for the comparison"),
```
Comment (eldiiar-duishenaliev) [inline pipelines/folioRancher/folioScheduledTesting/executeSprintBasedDataMigration/Jenkinsfile:14]: Specify descriptions please.

**Review by renatsafiulin — APPROVED** [review]

File: pipelines/folioRancher/folioTestingTools/folioSchemaCompare/Jenkinsfile [hunk 5]
```diff
@@ -6,7 +6,7 @@ import org.folio.rest.GitHubUtility
 import org.folio.Constants
 import groovy.json.JsonSlurperClassic
 
-@Library('pipelines-shared-library@RANCHER-741-Jenkins-Enhancements') _
+@Library('pipelines-shared-library@RANCHER-839') _
```
Comment (eldiiar-duishenaliev) [inline pipelines/folioRancher/folioTestingTools/folioSchemaCompare/Jenkinsfile:9]: don't forget to switch to 741 branch

File: pipelines/folioRancher/folioScheduledTesting/executeSprintBasedDataMigration/Jenkinsfile (hunk 4 above)
Comment (eldiiar-duishenaliev) [inline pipelines/folioRancher/folioScheduledTesting/executeSprintBasedDataMigration/Jenkinsfile:14]: switch to default value with R2-2023(Poppy)

**Review by eldiiar-duishenaliev — CHANGES_REQUESTED** [review]
Please review my comments.

File: pipelines/folioRancher/folioTestingTools/folioSchemaCompare/Jenkinsfile (hunk 5 above)
Comment (DmytrMoroz) [reply pipelines/folioRancher/folioTestingTools/folioSchemaCompare/Jenkinsfile:9]: done

**Pull request merged**